src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import static org.junit.Assert.*;
+
+import java.sql.Date;
+import java.util.Calendar;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.kuali.kfs.krad.exception.ValidationException;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractFile;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestDetailFileLine;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestEntryDetailFileLine;
+
+public class ConcurRequestExtractFileValidationServiceImplTest {
+    private ConcurRequestExtractFileValidationServiceImpl concurRequestExtractFileValidationService;
+    
+    @Before
+    public void setUp() throws Exception {
+        Logger.getLogger(ConcurRequestExtractFileValidationServiceImpl.class).setLevel(Level.DEBUG);
+        concurRequestExtractFileValidationService = new ConcurRequestExtractFileValidationServiceImpl();
+    }
+    
+    @After
+    public void tearDown() throws Exception {
+        concurRequestExtractFileValidationService = null;
+    }
+    
+    @Test
+    public void testHeaderAmountMatches() {
+        ConcurRequestExtractFile testFile = buildConcurRequestExtractFile();
```

Comment:
Maybe I am missing something here, but I think what we want to do with a unit test is to test that if we expect fileApprovedAmountsMatchHeaderApprovedAmount to return true when given input A, we are in fact getting true as result. If fileApprovedAmountsMatchHeaderApprovedAmount should return false when given input B then we do in fact get false as a result. The test as is written now, will always pass, no matter what fileApprovedAmountsMatchHeaderApprovedAmount returns.